composer.json

```diff
@@ -12,18 +12,18 @@
     }
   ],
   "require": {
-    "php": ">=5.5.9",
+    "php": "^7.1",
     "monolog/monolog": "^1.0",
-    "symfony/console": "^3.0",
```

Comment:
Supporting version 4 of Symfony should not imply to drop support for version 3, since the features provided by version 3 are still compatible with the current codebase. Please, update the constraints to support both versions instead (`^3.0|^4.0`).